packages/front-end/components/Features/SDKConnections/SDKLanguageLogo.tsx

```diff
@@ -176,23 +124,49 @@ export default function SDKLanguageLogo({
   showLabel = false,
   size = 25,
   titlePrefix = "",
+  version,
 }: {
   language: SDKLanguage;
   showLabel?: boolean;
   size?: number;
   titlePrefix?: string;
+  version?: string;
 }) {
   const { Icon, color, label } =
     languageMapping[language] || languageMapping["other"];
 
+  const versionOutdated = isSDKOutdated(language, version);
```

Comment:
Need to handle the case where version is undefined here

Reply:
oops forgot the `?:`